demo/src/examples/components/InputMoneyExample.js

```diff
@@ -38,6 +44,12 @@ const code = `<div>
     prefix="£"
   />
   <InputMoney
+    label='Amount without pennies'
+    placeholder='0'
+    prefix="£"
+    pennies={false}
```

Comment:
We could just invert the logic to use props in more conventional way, either empty or just pass props